tools/tikv-control.md

```diff
@@ -0,0 +1,164 @@
+raft state key: \001\002\000\000\000\000\000\000\000\002\002
+raft state: Some(hard_state {term: 307 vote: 5 commit: 314617} last_index: 314617)
+apply state key: \001\002\000\000\000\000\000\000\000\002\003
+apply state: Some(applied_index: 314617 truncated_state {index: 313474 term: 151})
+```
+
+### View the Region size
+
+Use the `size` command to view the Region size:
+
+```bash
+$ tikv-ctl --db /path/to/tikv/db size -r 2
+region id: 2
+cf default region size: 799.703 MB
+cf write region size: 41.250 MB
+cf lock region size: 27616
+```
+
+### Scan to view MVCC of a specific range
+
+The `--from` and `--to` options of the `scan` command accepts two escaped forms of raw key, and uses the `--show-cf` flag to specify the column families that you need to view.
+
+```bash
+$ tikv-ctl --db /path/to/tikv/db scan --from 'zm' --limit 2 --show-cf lock,default,write
+key: zmBootstr\377a\377pKey\000\000\377\000\000\373\000\000\000\000\000\377\000\000s\000\000\000\000\000\372
+         write cf value: start_ts: 399650102814441473 commit_ts: 399650102814441475 short_value: "20"
+key: zmDB:29\000\000\377\000\374\000\000\000\000\000\000\377\000H\000\000\000\000\000\000\371
+         write cf value: start_ts: 399650105239273474 commit_ts: 399650105239273475 short_value: "\000\000\000\000\000\000\000\002"
+         write cf value: start_ts: 399650105199951882 commit_ts: 399650105213059076 short_value: "\000\000\000\000\000\000\000\001"
+```
+
+### View MVCC of a given key
+
+Similar to the `scan` command, the `mvcc` command can be used to view MVCC of a given key.
+
+```bash
+$ tikv-ctl --db /path/to/tikv/db mvcc -k "zmDB:29\000\000\377\000\374\000\000\000\000\000\000\377\000H\000\000\000\000\000\000\371" --show-cf=lock,write,default
+key: zmDB:29\000\000\377\000\374\000\000\000\000\000\000\377\000H\000\000\000\000\000\000\371
+         write cf value: start_ts: 399650105239273474 commit_ts: 399650105239273475 short_value: "\000\000\000\000\000\000\000\002"
+         write cf value: start_ts: 399650105199951882 commit_ts: 399650105213059076 short_value: "\000\000\000\000\000\000\000\001"
+```
+
+In this command, the key is also the escaped form of raw key.
+
+### Print a specific key value
+
+To print the value of a key, use the `print` command.
+
+### Compact data manually
+
+Use the `compact` command to manually compact TiKV data. If you specify the `--from` and `--to` options, then the flag is also in the form of escaped raw key. You can use the `--db` flag to specify the RocksDB that you need to compact. The optional flag values are `kv` and `raft`.
+
+```bash
+$ tikv-ctl --db /path/to/tikv/db compact -d kv
+success!
+```
+
+### Set a Region to tombstone
+
+The `tombstone` command is usually used in circumstances where the sync-log is not enabled, and some write data of the Raft state machine is lost caused by power down.
```

Comment:
The `tombstone` command is usually used in circumstances where the sync-log is not enabled, and **some data** **written in** the Raft state machine is lost caused by power down.